packages/tasks/src/pipelines.ts

```diff
@@ -676,6 +676,11 @@ export const PIPELINE_DATA = {
 		color: "red",
 		hideInDatasets: true,
 	},
+	"document-retrieval": {
+		name: "Document Retrieval",
+		modality: "multimodal",
+		color: "yellow",
```

Comment:
```suggestion
		color: "yellow",
		hideInDatasets: true,
```
Don't think there are many datasets related to this task?

Reply:
there's a whole benchmark called ViDoRe and more datasets outside of this benchmark, and similar datasets exist in multiple languages so I would expect more datasets https://huggingface.co/spaces/vidore/vidore-leaderboard

Reply:
I'm not in favor of image-document-retrieval, it sounds a bit like images are involved separately and documents aren't images.